build/templates/session.py.mako

```diff
@@ -161,11 +167,15 @@ constructor_params = helper.filter_parameters(init_function['parameters'], helpe
 % if attributes:
 
 % endif
-    def __init__(self, repeated_capability_list, all_channels_in_session, library_interpreter, freeze_it=False):
+<% grpc_channel_param = " grpc_channel," if grpc_supported else "" %>\
+    def __init__(self, repeated_capability_list, all_channels_in_session, library_interpreter,${grpc_channel_param} freeze_it=False):
```

Comment:
It's generally bad to insert a parameter before an existing one, but I guess it's okay in this case, given that users shouldn't be initializing `_SessionBase`, themselves.